cmd/kube-proxy/app/server.go

```diff
@@ -237,6 +237,9 @@ func NewProxyServerDefault(config *options.ProxyServerConfig) (*ProxyServer, err
 		glog.V(0).Info("Tearing down pure-iptables proxy rules.")
 		iptables.CleanupLeftovers(iptInterface)
 	}
+	if err != nil && proxier == nil {
```

Comment:
Oh, i didn't know glog.Fatalf terminates the program, so the pr do not need, i close it, thanks